### PR TITLE
[Feature] Guide users to download artifacts to target base

### PR DESCRIPTION
**Pull request by wcchang1115**
**PR checklist**

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**What type of PR is this?**
Feature

**What this PR does / why we need it**:
- After preview change success, hint user to prepare base env.

**Which issue(s) this PR fixes**:
DRC-983

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

NONE

### CODE REVIEW

File: js/src/components/lineage/PreviewChangeView.tsx [hunk 1]
```diff
@@ -296,7 +305,10 @@ export function PreviewChangeView({
               icon={<VscFeedback />}
               variant={"ghost"}
               size={"md"}
-              onClick={() => feedbackToast(true)}
+              onClick={() => {
+                feedbackToast(true);
+                prepareEnvToast(true);
```
Comment (kentwelcome) [inline js/src/components/lineage/PreviewChangeView.tsx]: Show both feedback and prepare env toast when the user clicksthe feedback button seems weird.
`Give us feedback` button should only show a feedback toast.

**Comment by kentwelcome** [comment]
This guide is for Experiment 2, the production environment setup guide should only be shown when `single environment mode` is true. Not just show with the feedback popup on the preview change page.

**Review by kentwelcome — CHANGES_REQUESTED** [review]

File: js/src/components/lineage/PreviewChangeView.tsx (hunk 1 above)
Comment (wcchang1115) [reply js/src/components/lineage/PreviewChangeView.tsx]: ok, I will remove it.

**Review by kentwelcome — APPROVED** [review]
LGTM

**Pull request merged**